.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     timeout-minutes: 10
     strategy:
       matrix:
-        ENGINE_VERSION: [5.0.0, 5.1.0, 5.2.0, 5.3.0, 5.4.0, 5.5.0, 5.6.0, 5.6.1, latest]
```

Comment:
`latest` is already there for `5.7.0`. For me, it's not necessary to replace `5.6.1` which permit another test for that release.